cmd/api-server/main.go

```diff
@@ -242,6 +243,17 @@ func main() {
 				log.DefaultLogger.Infow("minio is not available, using default logs storage", "error", err)
 			}
 		}
+
+		// Run DB migrations
+		dbMigrator, err := dbmigrator.NewDbMigrator(db, "__migrations", cfg.TestkubeConfigDir)
+		ui.ExitOnError("Loading MongoDB migrations", err)
+		plan, _ := dbMigrator.Plan(ctx)
+		if plan.Total == 0 {
+			log.DefaultLogger.Info("No MongoDB migrations to apply.")
+		} else {
+			log.DefaultLogger.Info(fmt.Sprintf("Applying MongoDB migrations: %d rollbacks and %d ups.", len(plan.Downs), len(plan.Ups)))
+		}
+		ui.ExitOnError("Executing MongoDB migrations", dbMigrator.Apply(ctx))
```

Comment:
Should it be error or warning? Is it critical?

Reply:
Same as above